src/components/common/applicant-info-3.vue

```diff
@@ -147,19 +147,11 @@
                 <v-text-field :messages="messages['corpNum']"
                               :rules="corpNumRules"
                               :error-messages="corpNumError"
-                              validate-on-blur
-                              @blur="corpNumError = ''"
+                              @focus="corpNumError = ''"
                               :loading="loading"
                               filled
-                              v-on:update:error="setError"
-                              :hide-details="hideCorpNum"
                               :label="corpNumFieldLabel"
                               v-model="corpNum">
-                  <template v-slot:append>
```

Comment:
What about the similar code on applicant-info-2.vue ?

Reply:
I wonder if I need to wait for another ticket or do it together? I am not sure about the test case in applicant-info-2.vue.